swedish-oidc-profile.md

```diff
@@ -570,18 +565,15 @@ Section 2 of \[[OpenID.Registration](#openid-registration)\] defines a listing o
 
 The profile does not mandate OpenID Provider support for Dynamic registration of Relying Parties (clients). However, an OpenID Provider compliant with this profile MUST still be able to handle the client registration parameters specified in this section and in section 2 of \[[OpenID.Registration](#openid-registration)\]. This means that if an OpenID Provider registers Relying Parties by other means, it still needs to gather and maintain this client information.
 
```

Comment:
I would suggest upgrading the clarity of the requirement as "needs" is not a registered requirement.

Suggesting:
"This means that if an OpenID Provider registers Relying Parties by other means, it MUST gather and maintain all required client information parameters in the following table."